mmv1/products/bigqueryconnection/Connection.yaml

```diff
@@ -299,18 +299,48 @@ properties:
       - !ruby/object:Api::Type::String
         name: 'database'
         description:
-          Cloud Spanner database in the form `project/instance/database'
+          Cloud Spanner database in the form `project/instance/database'.
         required: true
       - !ruby/object:Api::Type::Boolean
         name: 'useParallelism'
         description:
-          If parallelism should be used when reading from Cloud Spanner
+          If parallelism should be used when reading from Cloud Spanner.
+      - !ruby/object:Api::Type::Integer
+        name: 'maxParallelism'
+        description:
+          Allows setting max parallelism per query when executing on Spanner independent compute
+          resources. If unspecified, default values of parallelism are chosen that are dependent on
+          the Cloud Spanner instance configuration. `useParallelism` and `useDataBoost` must be set
+          when setting max parallelism.
+        required_with:
+          - useDataBoost
+          - useParallelism
+      - !ruby/object:Api::Type::Boolean
+        name: 'useDataBoost'
+        description:
+          If set, the request will be executed via Spanner independent compute resources.
+          `use_parallelism` must be set when using data boost.
+        required_with:
+          - useParallelism
```

Comment:
This need to be the full path to the parameter.

see
https://github.com/GoogleCloudPlatform/magic-modules/blob/b7e0a8c9627221adef70ae7c6693821e55a26211/mmv1/products/secretmanager/Secret.yaml#L223